### PR TITLE
[ADF-5563][ADF-5565][ADF-5566]Removed !important from 'task-form','cloud-form-demo' and  'add-permission-panel' component

### PR DESCRIPTION
**Pull request by pkunduGL**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

!important is present in 'task-form','cloud-form-demo' and 'add-permission-panel' component

**What is the new behaviour?**
!important removed from 'task-form','cloud-form-demo' and 'add-permission-panel' component

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+    .adf-task-form {
     &-container {
         overflow: hidden;
     }
 
-    &-actions {
-        float: right;
-        padding-bottom: 25px !important;
-        padding-right: 25px !important;
+    &-actions{
```
Comment (rbahirsheth) [inline lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss:7]: Remove the extra line.

Comment (pkunduGL) [reply lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss]: resolved

**Review by rbahirsheth — APPROVED** [review]

File: lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss [hunk 2]
```diff
@@ -1,13 +1,12 @@
-.adf-task-form {
+    .adf-task-form {
```
Comment (MichalKinas) [inline lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss:1]: Indentation looks wrong in this file after your changes

File: lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
     &-container {
         overflow: hidden;
     }
 
-    &-actions {
+    &-actions{
```
Comment (MichalKinas) [inline lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss:6]:
```suggestion
    &-actions {
```

File: lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
         float: right;
-        padding-bottom: 25px !important;
-        padding-right: 25px !important;
-
+        padding-right: 25px;
+        padding-bottom: 25px;
```
Comment (MichalKinas) [inline lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss:9]: Missing empty space before next selector

File: lib/process-services/src/lib/task-list/components/task-form/task-form.component.html [hunk 5]
```diff
@@ -59,15 +59,17 @@ <h4>
...
-                <ng-template [ngTemplateOutlet]="taskFormButtons"></ng-template>
-                <button mat-button
-                    *ngIf="!isCompletedTask()" id="adf-no-form-complete-button"
-                    color="primary"
-                    [disabled]="canCompleteNoFormTask()"
-                    (click)="onCompleteTask()">
-                    {{'ADF_TASK_FORM.EMPTY_FORM.BUTTONS.COMPLETE' | translate}}
-                </button>
+            <mat-card-actions>
+                <div class="adf-task-form-actions">
```
Comment (MichalKinas) [inline lib/process-services/src/lib/task-list/components/task-form/task-form.component.html:63]: Why did you introduced additional div here?

File: lib/content-services/src/lib/permission-manager/components/add-permission/add-permission-panel.component.scss [hunk 6]
```diff
@@ -14,6 +14,16 @@ $search-result-height: calc(100% - 75px);
         &-search {
             display: none;
         }
+
+        .adf-list-option-item .mat-list-text {
```
Comment (nikita-web-ua) [inline lib/content-services/src/lib/permission-manager/components/add-permission/add-permission-panel.component.scss:18]: please try not to use Material classes

File: lib/process-services/src/lib/task-list/components/task-form/task-form.component.html (hunk 5 above)
Comment (pkunduGL) [reply lib/process-services/src/lib/task-list/components/task-form/task-form.component.html:63]: In order to eliminate the necessity of relying on the internal material classes, we created a wrapper div and added styling in accordance with our specifications.

File: lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss (hunk 4 above)
Comment (pkunduGL) [reply lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss:9]: Done

File: lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss (hunk 2 above)
Comment (pkunduGL) [reply lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss]: Done

File: lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss (hunk 3 above)
Comment (pkunduGL) [reply lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss]: Done

File: lib/content-services/src/lib/permission-manager/components/add-permission/add-permission-panel.component.scss (hunk 6 above)
Comment (pkunduGL) [reply lib/content-services/src/lib/permission-manager/components/add-permission/add-permission-panel.component.scss]: Done

File: lib/process-services/src/lib/task-list/components/task-form/task-form.component.html (hunk 5 above)
Comment (MichalKinas) [reply lib/process-services/src/lib/task-list/components/task-form/task-form.component.html:63]: I can't see any styles related to this class, is it used anywhere?

Comment (pkunduGL) [reply lib/process-services/src/lib/task-list/components/task-form/task-form.component.html:63]: In line number 6 of (lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss)

**Comment by nikita-web-ua** [comment]
Check please if all style changes correctly applied in apps, for example as I see in ACA checkboxes are visible which should be hidden 
```
mat-pseudo-checkbox {
            display: none;
        }
```

Screenshot 2023-12-21 at 12 37 11

**Comment by pkunduGL** [comment]
> Check please if all style changes correctly applied in apps, for example as I see in ACA checkboxes are visible which should be hidden
> 
> ```
> mat-pseudo-checkbox {
>             display: none;
>         }
> ```
> 
> Screenshot 2023-12-21 at 12 37 11

Issue fixed

**Review by AleksanderSklorz — APPROVED** [review]

**Review by AleksanderSklorz — APPROVED** [review]

**Review by swapnil-verma-gl — APPROVED** [review]

**Pull request merged**